scripts/import_db_from_json.sh

```
@@ -28,13 +28,25 @@ done
 
 for collection in "${collections[@]}"
 do
+  IMPORT_FILE="/import/${collection}.json"
+  
+  # Check if the file contains only an empty array
+  CONTENT=$(docker exec "$CONTAINER" cat "$IMPORT_FILE")
+  if [ "$CONTENT" == "[]" -o "$CONTENT" == "" ]; then
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)